src/main/java/de/tum/in/www1/artemis/domain/metis/conversation/Channel.java

```diff
@@ -71,6 +71,14 @@ public class Channel extends Conversation {
     @NotNull
     private Boolean isArchived;
 
+    /**
+     * Channels, that are meant to be seen by all course members by default, even if they haven't joined the channel yet, can be flagged with is_course_wide=true.
+     * A conversation_participant entry will be created on the fly for these channels as soon as an entry is needed.
+     */
+    @Column(name = "is_course_wide")
+    @NotNull
+    private Boolean isCourseWide = false;
```

Comment:
You should be able use `boolean` here since the column is not nullable.

Reply:
You are right, I changed the type.